<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_right">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/chevrons/images/chevron_r.png</normaloff>:/chevrons/images/chevron_r.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_left">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/chevrons/images/chevron_l.png</normaloff>:/chevrons/images/chevron_l.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>560</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lbl_new_">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menuAdditional">
     <property name="title">
      <string>Период переключения</string>
     </property>
     <addaction name="action_0sec"/>
     <addaction name="action_1sec"/>
     <addaction name="action_5sec"/>
     <addaction name="action_10sec"/>
    </widget>
    <addaction name="action_up_windows"/>
    <addaction name="action_use_resources"/>
    <addaction name="action_choose_dir"/>
    <addaction name="menuAdditional"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_up_windows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Поверх всех окон</string>
   </property>
  </action>
  <action name="action_use_resources">
   <property name="text">
    <string>Фото из файла ресурса</string>
   </property>
  </action>
  <action name="action_choose_dir">
   <property name="text">
    <string>Фото из выбранной папки</string>
   </property>
  </action>
  <action name="action_0sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="action_1sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1 секунда</string>
   </property>
  </action>
  <action name="action_5sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>5 секунд</string>
   </property>
  </action>
  <action name="action_10sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>10 секунд</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
